packages/ui/react-ui-theme/src/styles/components/input.ts

```diff
@@ -112,7 +112,7 @@ export const inputCheckbox: ComponentFunction<InputStyleProps> = ({ size = 5 },
   mx(
     getSize(size),
     'shrink-0 flex items-center justify-center rounded text-white',
-    'bg-neutral-200 dark:bg-neutral-700 aria-checked:bg-primary-600 dark:aria-checked:bg-primary-600 aria-[checked=mixed]:bg-primary-600 dark:aria-[checked=mixed]:bg-primary-600',
+    'bg-neutral-100 dark:bg-neutral-800 aria-checked:bg-primary-600 dark:aria-checked:bg-primary-600 aria-[checked=mixed]:bg-primary-600 dark:aria-[checked=mixed]:bg-primary-600',
```

Comment:
This makes checkboxes invisible in dark mode within chrome surfaces.

Reply:
```suggestion
    'bg-neutral-100 dark:bg-neutral-700 aria-checked:bg-primary-600 dark:aria-checked:bg-primary-600 aria-[checked=mixed]:bg-primary-600 dark:aria-[checked=mixed]:bg-primary-600',
```
Would avoid these disappearing completely in certain situations.

Reply:
Maybe unselected checkboxes should consistently lighten/darken their surface, in which case we could look into using partly-transparent color tokens instead of fully opaque ones.